package.json

```diff
@@ -22,6 +22,9 @@
     "url": "git+https://github.com/mozilla/tab-split.git"
   },
   "scripts": {
+    "start": "npm-run-all lint build run",
```

Comment:
Not sure if anybody is rocking Windows (or if Win 10.x supports a real Unixy language yet w/ PowerShell, Bash, etc), or if we need to fall back to something like npm [**mkdirp**](https://www.npmjs.com/package/mkdirp) for cross OS greatness.

Reply:
Supporting windows sounds good to me. I don't use it so can't test it.

Reply:
The other alternative is just check in an empty ./Profile directory with a nested .gitignore file which ignores everything. That way we aren't constantly trying to constantly re-run `mkdir -p ./Profile` (which would get called twice on `npm start`).

Reply:
But i'm also happy to just merge this as-is and we can over optimize later when the Bugzilla bug is fixed.

Reply:
Let's just add a bug about supporting windows.

Reply:
https://github.com/mozilla/tab-split/issues/54